### DISCUSSION TITLE
Force Musescore to recover previous session

### DISCUSSION THREAD

**Discussion by WilliamDiakite** [musescore/MuseScore#14776] [Q&A]
Musescore crashed every time I tried to save my file. I had to force quit the app several times and reopen it using the dialog box to recover last session. Because I was still unable to save my file without Musescore carshing I tried to reopen it without restoring the last session. Eventually I lost all my work without ever being able to save it... 

Is their a way to recover that file by forcing Musescore to restaure a previous session? 

Using Musescore 3 on MacOs 12.5 (Montmery)

**Comment by ixd704** [comment]
After it crashed twice, I would have exported the file as MIDI or some other format I could import into a new file as that way you would have possibly recovered parts of the file that were not corrupted.  I would have also made a copy of the file before opening it without recovery.  I found that opening a file that had issues with the beta 4 version, fixed issues I had with a percussion track not exporting or being listed for export.  I'm not sure if this would have worked for you, but possibly.  Did you check if the Mac had a previous version saved by the OS?  If you have timemachine enabled, you get multiple copies of your files created going back to the first save.
